src/netius/servers/proxy.py

```
@@ -872,6 +938,20 @@ def _on_prx_headers(self, client, parser, headers):
         # to be used to send the headers (and status line) to the client
         connection = self.conn_map[_connection]
 
+        # an informational response is an interim one, it carries no payload
+        # and is terminated by the end of its headers, so it's relayed as is
+        # and none of the framing decisions apply to it, note that a client
+        # with no support for it must never be presented with one
+        code = int(code_s)
+        if code < 200 and not code == 101:
+            if connection.parser.version < netius.common.HTTP_11:
```

Pylint (4.0.7)

[error] Module 'netius.base.common' has no 'HTTP_11' member

(E1101)